bounties_api/std_bounties/client_helpers.py

```diff
@@ -47,10 +48,25 @@ def map_bounty_data(data_hash, bounty_id):
         ipfs_hash = 'invalid'
 
     data = json.loads(data_JSON)
-    metadata = data.get('meta', {})
+
+    meta = data.get('meta', {})
+
     if 'payload' in data:
         data = data.get('payload')
 
+    metadata = data.get('metadata', {})
+
+    if metadata.get('experienceLevel', '').lower().strip().capitalize() in rev_mapped_difficulties:
+        metadata.update({
+            'experienceLevel': rev_mapped_difficulties.get(
+                metadata.get('experienceLevel').lower().strip().capitalize()
+            )
+        })
+    else:
```

Comment:
@c-o-l-o-r I think this reads more cleanly and uses constants.

```
experienceLevel = BEGINNER
inputExperienceLevel = metadata.get('experienceLevel', '').lower().strip().capitalize()
if inputExperienceLevel:
  experienceLevel = rev_mapped_difficulties.get(inputExperienceLevel, BEGINNER)
metadata.update({ 'experienceLevel': experienceLevel })
```

Reply:
Alternatively - with the DEFAULT behavior - and not blank on the django field - (I'm not sure if this will work) - you could even do: 
```
formattedExperienceLevel = metadata.get('experienceLevel', '').lower().strip().capitalize()
metadata.update({ 'experienceLevel': rev_mapped_difficulties.get(inputExperienceLevel, '') })
```
But that's only if django turns a blank string '' to the default automatically ( I always forget)

Reply:
you're totally right, that other thing was terrible -- no idea what I was thinking.